dpe/src/dpe_instance.rs

```diff
@@ -422,6 +414,48 @@ impl Iterator for FlagsIter {
     }
 }
 
+#[repr(C)]
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+#[cfg_attr(test, derive(zerocopy::AsBytes, zerocopy::FromBytes))]
+pub struct ContextHandle(pub [u8; ContextHandle::SIZE]);
+
+impl ContextHandle {
+    pub const SIZE: usize = 16;
+    const DEFAULT: [u8; Self::SIZE] = [0; Self::SIZE];
+
+    /// Returns the default context handle.
+    pub const fn default() -> ContextHandle {
+        ContextHandle(Self::DEFAULT)
+    }
+
+    /// Whether the handle is the default context handle.
+    pub fn is_default(&self) -> bool {
+        self == &Self::default()
```

Comment:
nit: Is this doing a copy? Might be better to just compare `self.0` with `DEFAULT`. But not a big deal either way.

Reply:
I don't think so... but, I'm not sure. I think your suggestion is more explicit about what it is doing anyway. I changed it.